src/app/routes/profile/containers/editprofile.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react'
+import { push } from 'react-router-redux'
+import store from '../../../store/configureStore'
 import countries from '../../../components/forms/commons/countries'
-import phone_types from '../../../components/forms/commons/phonetypes'
-import address_types from '../../../components/forms/commons/addresstypes'
+import { phoneTypes, addressTypes }  from '../../../components/forms/commons/address_phone_types'
 import UiValidate from '../../../components/forms/validation/UiValidate'
 import JarvisWidget from '../../../components/widgets/JarvisWidget'
+import { bigBox } from "../../../components/utils/actions/MessageActions";
+import '../../../../../node_modules/jquery.maskedinput/src/jquery.maskedinput.js'
+
+// Used for checking file (photo)
+const FILE_CONSTANTS = {
+    selected: 1,
+    index: 0,
+    sizelimit: 100000   // (bytes) this will change
+}
+const ALERT_TIMEOUT = 6000  // 6 seconds
```

Comment:
This seemed like a reasonable time for the user to see the alert, and dismiss it on their own or allow it to dismiss itself. Resulted to 6 seconds after testing shorter and longer times